lib/src/summarize.dart

```diff
@@ -4,12 +4,51 @@
 
 library dart_pad.summarize;
 
+import '../services/dartservices.dart';
 /// Instances of this class take string input of dart code and output a text 
 /// description ofthe code's size, packages, and other useful information
 class Summarizer {
   
-  String summarize(String input) {
-    return "foo";
+  String summarize(SourceRequest input, AnalysisResults result) {
+    if (input == null || result == null) {
+      return "Summarizer has broken!.";
+    }
+    String summary = '<pre><code><b>Summary (Under Development)</b><p/>';
+    summary += '${_linesCode(input)} lines of code used. <p/>';
+    bool hasErrors = result.issues.any((issue) => issue.kind == 'error');
+    bool hasWarnings = result.issues.any((issue) => issue.kind == 'warning');
+    if (hasErrors) summary += 'Errors are present.<p/>';
+    if (hasWarnings) summary += 'Warnings are present.<p/>';
+    summary += _languageFeatures(result);
+    summary += '${result.packageImports.length} ';
+    summary += 'Package Imports: ${result.packageImports.toString()}. <p/>';
+    summary += '${result.resolvedImports.length} ';
+    summary += 'Resolved Imports: ${result.resolvedImports.toString()}. <p/>';
+    summary += 'List of ${result.issues.length} Errors: <ul>';
+    for (AnalysisIssue issue in result.issues) {
+      summary += '<li>${_condenseIssue(issue)}</li>';
+    }
+    summary += '</ul></code></pre>';
+    return summary;
+  }
+  
+  ///Outputs the language features used
```

Comment:
It's not critical to have doc comments for all methods. It's very nice to have it for major, public methods. Small, private ones are often self-documenting.

nit for formatting: Please leave a space after the `/// ` and end the comment with a period (it's a full sentence, not a sentence fragment).